build/tasks/AddMetapackageReferences.cs

```diff
@@ -37,21 +41,36 @@ public override bool Execute()
 
             // Items
             var itemGroupElement = xmlDoc.CreateElement("ItemGroup");
-            Log.LogMessage(MessageImportance.Normal, $"Runtime store will include the following packages");
+            Log.LogMessage(MessageImportance.High, $"Runtime store will include the following packages");
+
             foreach (var package in metapackageArtifacts)
             {
                 var packageName = package.GetMetadata("Identity");
                 var packageVersion = buildArtifacts
                     .Single(p => string.Equals(p.PackageInfo.Id, packageName, StringComparison.OrdinalIgnoreCase))
                     .PackageInfo.Version.ToString();
-                Log.LogMessage(MessageImportance.Normal, $" - Package: {packageName} Version: {packageVersion}");
+                Log.LogMessage(MessageImportance.High, $" - Package: {packageName} Version: {packageVersion}");
+
+                var packageReferenceElement = xmlDoc.CreateElement("PackageReference");
+                packageReferenceElement.SetAttribute("Include", packageName);
+                packageReferenceElement.SetAttribute("Version", packageVersion);
+
+                itemGroupElement.AppendChild(packageReferenceElement);
+            }
+
+            foreach (var package in externalArtifacts)
+            {
+                var packageName = package.GetMetadata("Identity");
+                var packageVersion = package.GetMetadata("Version");
+                Log.LogMessage(MessageImportance.High, $" - Package: {packageName} Version: {packageVersion}");
```

Comment:
MessageImportance.Normal

Reply:
It was actually a little tricky to track down what the issue was with the missing packages. I would rather like to see which references were added in the build log. This is why I set all of these reference-adding tasks to use a high message importance.